src/hotspot/share/gc/shenandoah/shenandoahVerifier.cpp

```diff
@@ -602,11 +602,11 @@ class ShenandoahVerifierMarkedRegionTask : public AbstractGangTask {
 
 class VerifyThreadGCState : public ThreadClosure {
 private:
-  const char* _label;
-  char _expected;
+  const char* const _label;
+         char const _expected;
```

Comment:
Why can't _expected not be const char* const too? Are we really messing with the string on the way?

Reply:
`_expected` is not `const char*`, it is `const char`, where `const` is placed near the field identifier for consistency.

Reply:
@rkennke Still good with this explanation?

Reply:
Yes, thank you! Stupid me overlooked the missing * :-)